pkg/tiller/release_rollback.go

```diff
@@ -151,11 +152,16 @@ func (s *ReleaseServer) performRollback(currentRelease, targetRelease *release.R
 		}
 	}
 
+	// update the current release
+	s.Log("superseding previous deployment %d", currentRelease.Version)
+	currentRelease.Info.Status.Code = release.Status_SUPERSEDED
+	s.recordRelease(currentRelease, true)
+
+	// Supersede all previous deployments, see issue #2941.
 	deployed, err := s.env.Releases.DeployedAll(currentRelease.Name)
-	if err != nil {
+	if err != nil && !strings.Contains(err.Error(), "not found") {
```

Comment:
Is there any way to the call to `DeployedAll` can return an error containing the string "not found"? Seems like the implementation of `DeployedAll` already checks for this condition and returns an error with a different message in this case:
https://github.com/helm/helm/blob/77924fec911168bc490054c67904f1244ab475e3/pkg/storage/storage.go#L152-L154

Reply:
Great catch. I was thinking it was wrapping the original error. I will update the pr.